MARKETPLACE_TOOLS_TAG

```diff
@@ -1 +1 @@
-0.8.0
+0.9.2
```

Comment:
lineend

Reply:
Done.

Reply:
Replaced - it turns out that the newline causes this tag not to be recognized by CloudBuild. You can replace later if you figure out how to make it work :)